pygame2_extra_deps.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "pygame2_extra_deps",
+  "buildsystem": "simple",
+  "build-commands": [],
+  "modules": [
+    {
+      "name": "libmodplug_sdl_vendored",
+      "buildsystem": "cmake",
+      "config-opts": [ "-DBUILD_SHARED_LIBS=ON", "-DFOO=BAR" ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/libsdl-org/libmodplug",
+          "commit": "f3173c24dcce28d54429099c555665a59d729546"
+        }
+      ]
+    },
+    {
+      "name": "FluidSynth",
```

Comment:
fluidsynth is in the shared modules https://github.com/flathub/shared-modules/ and should be prefered

Reply:
Ahh. I missed that folders exist for grouping together things less related than SDL, its supplementary libraries, and their various versions.

Next time, I'll do a proper search bar search before assuming something isn't in there.